api/proto/auth/client.go

```diff
@@ -0,0 +1,186 @@
+package auth
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+	"sync/atomic"
+
+	"github.com/gravitational/roundtrip"
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+var log = logrus.WithFields(logrus.Fields{
+	trace.Component: teleport.ComponentAuth,
+})
```

Comment:
Different component now.

Reply:
Shoud this be `ComponentGRPC`, or should I add `ComponentAPI`?